micall_basespace.py

```diff
@@ -326,10 +326,8 @@ def link_samples(run_path: str, data_path: str, is_denovo: bool):
     shutil.rmtree(data_path, ignore_errors=True)
     makedirs(data_path)
 
-    results_path = os.path.join(run_path, 'Results', 'basespace')
```

Comment:
Sure -- should we make a duplicate copy of all the results in that case?

Reply:
I don't think that's needed right now. I only care about using a symlink when I'm running it on my workstation in the host system. Under docker, it can just write the files to the local directory.

Reply:
Sorry I just remembered something -- the symlink doesn't fail at all, but the resulting symlink is (or seems to be) non-functional outside of the Docker container.  This is what seemed to happen:

* I mounted `foo/` on the host as the `/input` directory in the container.
* I mounted `bar/` on the host as `/data` in the container.
* The resulting link on the host looked like `bar/output@ -> /input/Results/basespace`

Reply:
Interesting. That makes sense, I guess - the paths are meaningless in the host system.
How about using hard links? That should either be consistent across the two systems, or fail under docker. If it fails, fall back to creating a local copy.
If you can't make it fail gracefully, please add a command-line option to create the symlink back to the MiSeq run folder, and default it to off. When I run it on my host system, I'll just specify the option.

Reply:
Sounds good, I'll give that a try.  To clarify, when you say "fall back to creating a local copy" do you mean "copy the contents of `[input directory]/Results/basespace` to `[output directory]/output`"?

Reply:
In your example above, it would mean creating `bar/output` as a simple directory, and writing the results there. In other words, if you can't create a link, create a simple directory. Either way, just write the output files there.